app/models/ability.rb

```diff
@@ -3,6 +3,7 @@ class Ability
 
   def initialize(user)
     user ||= User.new
-    can :destroy, Recipe, user:
+    can :destroy, Recipe, user: user
+    can :destroy, Food, user:
   end
```

Comment:
SyntaxError is caused because `user:` instead of  `user: user` can only be used after ruby 3.1 and midline or inside parentessis

Reply:
Yes I actually had it as ```user: user``` but for some reason, rubocop marks it as an error, should I ignore that linter error?

Reply:
no, you can use `'user' => user`